model/src/main/resources/db-update/20000-from-1.8.x-to-2.0.0.sql

```diff
@@ -188,3 +188,34 @@ BEGIN transaction;
     ALTER TABLE buildrecord ADD CONSTRAINT fk_buildrecord_norebuildcause
     FOREIGN KEY (norebuildcause_id) REFERENCES buildrecord(id);
 COMMIT;
+
+
+BEGIN transaction;
+
+    -- Verify that there are no duplicate data
+    do $$
+    declare
+        total_dupes integer;
+    begin
+        SELECT count(*)
+        INTO total_dupes
+        FROM (SELECT built_artifact_id FROM build_record_built_artifact_map GROUP BY built_artifact_id HAVING count(*) > 1) AS agg;
+
+        assert total_dupes = 0;
+    end$$;
+
+    -- Add new column to artifact table
+    ALTER TABLE artifact ADD column buildrecord_id int4;
+    CREATE INDEX idx_artifact_buildrecord ON artifact (buildrecord_id);
```

Comment:
Add the Index also to Artifact model here https://github.com/project-ncl/pnc/blob/b9b48674b743e92b36ff57960b820b0f3d26d2d7/model/src/main/java/org/jboss/pnc/model/Artifact.java#L71